composer.json

```diff
@@ -28,7 +28,7 @@
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.0.0",
-        "phpunit/phpunit": "^4.8.31"
+        "phpunit/phpunit": "^4.8.35 || ^5.7 || ^6.0"
```

Comment:
One more thing, how about making it more obvious which minor versions we are fine with?

```diff
-"phpunit/phpunit": "^4.8.35 || ^5.7 || ^6.0"
+"phpunit/phpunit": "^4.8.35 || ^5.7.0 || ^6.0.0"
```

This way it is very accurately documented.